zhaquirks/tuya/__init__.py

```diff
@@ -436,6 +436,14 @@ async def write_attributes(self, attributes, manufacturer=None):
             cmd_payload.function = 0
             cmd_payload.data = Data.from_value(record.value.value)
 
+            _LOGGER.debug(
```

Comment:
You can use `self.debug` on a cluster or device and it would include all identifying information, so you can omit nwk, endpoint and cluster

Reply:
I've copied that originally from TuyaThermostatCluster
Wanna to keep same approach.
Tested locally on my setup.